src/app/App.js

```diff
@@ -1,15 +1,20 @@
 import React from "react"
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
```

Comment:
You dont use Switch, so you can delete it!

Reply:
yeah thanks deleted that!